src/app.html

```diff
@@ -58,8 +58,43 @@
   </div>
 
   <div class="main" if.bind="activeForm">
-    <compose class="form" view-model.bind="activeForm"></compose>
-    <div class="preview" textcontent.one-way="currentFormJSON"></div>
+    <div class="about" if.bind="activeForm === 'about'">
+      <div class="about-inner">
+        <h3>What is OpenAPI designer?</h3>
+        <p>
+          OpenAPI designer allows you to create and edit API design files.
+          It is a a form based editor for designing APIs.
+          It currently supports the <a href="http://swagger.io/">OpenAPI/Swagger</a> version 2.0.
+        </p>
+
+        <h3>Why use OpenAPI designer?</h3>
+        <p>
+          Developers can get started more easily if they can view an API design document and can try out the API calls.
+          API designers can share the design file and get feedback from other stakeholders. This facilitates collaborative API design process.
+          Coding skills and understanding of JSON/YAML is not required for using the OpenAPI designer.
```

Comment:
New paragraph here. The first paragraph is about collaboration between Developers and API designers. The second paragraph is about coding skills and the UI.